app/src/components/organisms/SurveyForm.tsx

```diff
@@ -248,6 +283,26 @@ function SurveyForm(props: { formData: IClassData[] }) {
             <TermOptions handleChange={handleTerm} term={'summer'} label="Summer" prefs={termPrefs.summer} />
           </Stack>
 
+          <Stack direction="row" spacing={2}>

```

Comment:
Move this to just below the course list (top of the section), since it has more to do with course selection than anything else